### ISSUE TITLE
Version change from 0.4.1 to >=0.4.2 results in missing bar in plot

### ISSUE THREAD

**Issue by gSilas** [jnothman/UpSetPlot#124]
With the same script plotting four different categories produced by UpSetPlot==0.4.1 results in the expected UpSet plot with a bar for each category:
![upset_sequences_0 4 1](https://user-images.githubusercontent.com/9419793/121391115-4f4cd700-c94e-11eb-91a8-93644968f3de.png)
With UpSetPlot=>0.4.2 the UpSet plot does not display the bar for the unique elements in the category with the most unique values but the count is still shown: 
![upset_sequences_0 4 3](https://user-images.githubusercontent.com/9419793/121391096-49ef8c80-c94e-11eb-8ee0-374703ec5169.png)
The Biogas category is missing.
The bug can also be reproduced with this simple code example:
```
upset_data = upsetplot.from_contents({'cat1': [0], 'cat2': [1], 'cat3': [2], 'cat4': [3]})
upsetplot.plot(data=upset_data, show_counts='%d')
plt.savefig('test.png')
```
**Expected plot (0.4.1) :**
![0.4.1 plot](https://user-images.githubusercontent.com/9419793/121393043-3b09d980-c950-11eb-8e73-f1de5dee783b.png)

**Plot produced by 0.4.3:**
![0.4.3 plot](https://user-images.githubusercontent.com/9419793/121393142-5b399880-c950-11eb-8122-258572b3bec3.png)

**Comment by jnothman** [comment]
Oh dear, it seems we need more test cases! Thanks for the report

**Comment by jnothman** [comment]
Thoughts @huangy6? The proposed patch reverts the change in autoscale setting

**Comment by jnothman** [comment]
Sorry, I was a bit hasty in my review and release of #106. I've got a proper fix in #126
